pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[tool.poetry]
+name = "hvac"
+version = "1.0.0"
+description = "HashiCorp Vault API client"
+readme = "README.md"
+authors = ["Ian Unruh <ianunruh@gmail.com>, Jeffrey Hogan <jeff.hogan1@gmail.com>"]
+maintainers = ["Brian Scholer <brian@briantist.com>", "Colin McAllister <colinmca242@gmail.com>", "Mike Adams <Mike.Adams@kyndryl.com>"]
```

Comment:
Would it be ok to remove my email address and use my GitHub ID `@briantist`? Or maybe the committer email I use if an email address is necessary? `1260690+briantist@users.noreply.github.com`

Reply:
That's fine with me, I added the committer email.